src/test/e2e/services/access-service.e2e.test.ts

```diff
@@ -363,6 +363,9 @@ test('should remove CREATE_FEATURE on default environment', async () => {
         permissions.CREATE_FEATURE,
         '*',
     );
+    expect(
+        await accessService.hasPermission(user, CREATE_FEATURE, 'some-project'),
```

Comment:
If these are global permissions, perhaps we should check `default` project instead.

```suggestion
        await accessService.hasPermission(user, CREATE_FEATURE, 'default'),
```

Reply:
Alternatively, perhaps 'some-project' does not exist in the context of the test, so permissions for this project always return false.